src/actions.js

```diff
@@ -2,7 +2,7 @@ import { getApplicationSchema, saveValues as save } from './api';
 import { ACTION_TYPES } from './constants';
 import config from './config/config.json';
 
-const notificationConfigForBundle = (bundleName) =>
+export const notificationConfigForBundle = (bundleName) =>
```

Comment:
So, for the sake of having code where it belongs, I think we should move out this function to a separate place. 

This file have the redux actions, maybe put it on utilities: https://github.com/RedHatInsights/user-preferences-frontend/blob/master/src/Utilities/functions.js